### ISSUE TITLE
Ads not collected on myip.com

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#1906]
https://www.myip.com/
![image](https://user-images.githubusercontent.com/737638/127725024-7e853abd-4954-45bc-9966-7a01d68e5448.png)

**Comment by mneunomne** [comment]
@dhowe in my location the ad that appears are the type `GoogleActiveViewElement`:

![printprint](https://user-images.githubusercontent.com/4967860/129108661-084ca643-ed4b-4868-a7b3-3ec46d9c960c.PNG)


I realized that the function `findGoogleResponsiveDisplayAd` was rather outdated in the parser.js. I was able to update it to be able to correctly collect this types of ads, which are many of the cases apparently.

![collections](https://user-images.githubusercontent.com/4967860/129108693-db98f95a-4ca5-4af8-a94e-19d9c5faef38.PNG)

--------------------------------------------

The thing is for these types of ads to be rendered I needed to allow multiple things.

For chrome I need to allow the `no-floc` uBlock rule, with this exception rule:
```
#@#+js(no-floc)
```
Which I took reference from this issue https://github.com/uBlockOrigin/uBlock-issues/issues/1553 . We would need to think if we want to have this or not

--------------------------------------

and also these exceptions, including the last two that are more problematic:
```
@@||googleads.g.doubleclick.net^$subdocument
@@||c.statcounter.com^$ping
@@||secure.statcounter.com^$script

#@#.adsbygoogle
#@#.ads_1
```
It seems that even though the last two are cosmetic rules, google doesn't render the ads if the parent element has all the hidding property that we inject, we would need to figure a way around that (perhaps injecting the hiding properties after running the parser?). 

----------------------------------------

### To sum up, if we want to be able to collect these googleads we need to do some workarounds  for them to be rendered and then collected.

**Comment by dhowe** [comment]
excellent work @mneunomne -- as Google is still the largest server of ads, I think we do want to do the work to render/collect them...

**Comment by dhowe** [comment]
@mneunomne status on this?

**Comment by mneunomne** [comment]
I am able to collect them with the big work around I mentioned above. The issue is how to hide the row where these ads used to be without making a cosmetic filter for the element itself ".adsbygoogle"

![collelcteddd](https://user-images.githubusercontent.com/4967860/133931238-e8d0afa3-803d-451b-b515-1e35ec71dbce.PNG)

Here I can simply create a filter for the parent of ".adsbygoogle" but I am not sure that would work on all cases (which are many many that use .adsbygoogle).

Will think of a solution and get back here as soon as I figure it out.

**Comment by mneunomne** [comment]
Made some more improvements and it seems to be collecting quite a few at least in my region.
Created this separate issue so we can document more cases like this: https://github.com/dhowe/AdNauseam/issues/1932

**Comment by mneunomne** [comment]
Fixed, tested on FF and Chrome
